.github/workflows/rebase-insiders.yml

```diff
@@ -61,6 +61,8 @@ jobs:
       - name: push changes
         run: |
           set -e
+          git add ./build/artifacts/artifacts.lock.yaml
+          git commit -m "Update artifacts.lock.yaml" --signoff
```

Comment:
I guess generate.sh script should be run for every rebasing against upstream
is it ^ correct?

if so - please add the corresponding steps to the readme https://github.com/che-incubator/che-code?tab=readme-ov-file#fixing-the-rebase-insiders-workflow
Rebasing against upstream is often done manually because of conflicts - so - person who provides alignment with upstream should run generate.sh script manually as well